### PR TITLE
tests(lantern): rebase

### PR DESCRIPTION
**Pull request by connorjclark**
What is looks like today:

image

___

This is with the new traces, comparing with the old data. I wouldn't give this too much concern. See the side-by-side instead.

lantern-rebase

After rebase:

image

Side by side (before - after):

image

### CODE REVIEW

File: lighthouse-core/scripts/lantern/download-traces.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ rm -rf lantern-data/
 mkdir -p lantern-data/ && cd lantern-data
 echo $VERSION > version
 
-curl -o lantern-traces.tar.gz -L https://storage.googleapis.com/lh-lantern-data/lantern-traces-$VERSION.tar.gz
+curl -o golden-lantern-traces.zip -L https://storage.googleapis.com/lh-lantern-data/golden-lantern-traces-$VERSION.zip
```
Comment (connorjclark) [inline lighthouse-core/scripts/lantern/download-traces.sh]: renamed file b/c we are also keeping the big zip files in this bucket, and I wanted to reuse the naming that the lantern scripts use.

**Comment by patrickhulce** [comment]
wowzah am I reading this right that the median error for LCP is 108% (though it seems like the p50 and p90 are flipped and it should be 78%?, still...)

**Comment by connorjclark** [comment]
> would be nice to get a before shot in the PR that has the full dataset for posterity instead of just a couple overlap URLs :)

good point. also updated the side-by-side ..

> just curious do these traces also have CLS events for us too? I think the plan is to not have a separate lantern impl for it but it would be nice to have a corpus of those in our golden set already

none of these traces have `LayoutShift` events. Must not be in stable yet.

**Comment by connorjclark** [comment]
> wowzah am I reading this right that the median error for LCP is 108% (though it seems like the p50 and p90 are flipped and it should be 78%?, still...)

yeah its high. "should" be? It's backwards for sure, but I don't think the percentile maths is wrong or anything.

**Comment by patrickhulce** [comment]
> "should" be? It's backwards for sure, but I don't think the percentile maths is wrong or anything.

"should be" meaning if it's just flipped then the p50 error is 78%, not 108%.

**Pull request merged**